### ISSUE TITLE
Cannot "Copy and Paste" any info from Franz

### ISSUE THREAD

**Issue by hannatonegawa** [meetfranz/franz#975]
### Expected Behavior
I try to copy-and-paste (using either Command-C and Command-V or using "Edit->Copy/Paste"), but neither works for any of the applications I use on Franz (Gmail, Trello, FB Messenger, Slack). I cannot copy and paste any text from one application to the other, or even within one application itself.

### Current Behavior
Whenever I highlight new content to try and copy and paste into another text field within Franz, the last thing I copied from other services (Google Chrome, etc.) appears instead.

### Steps to Reproduce (for bugs)
1.
2.
3.
4.

### Context
I think this is simply a bug fix...? This copy-and-paste function works well on a Windows computer and not on my Mac computer....

### Your Environment
* Franz Version used: Version 5.0.0-beta.18 (5.0.0-beta.18.903)
* Operating System and version: macOS High Sierra Version 10.13.4

**Comment by noctarius** [comment]
Same here

**Comment by roberterdin** [comment]
Same. This is also quite a major security issue since you can't use passwords generated with a password manager. (unless you want to type 30 characters, digits and symbols)

**Comment by bartmeuris** [comment]
Yeah - especially the password pasting is problematic. Although this looks like a promising piece of software, I won't create an account as long as this isn't possible...

**Comment by shalafi83** [comment]
same here, only "right click -> copy" works, keyboard shortcuts don't.

**Comment by bartmeuris** [comment]
Right click doesn't work on the create new account window...

**Comment by kris-savoia** [comment]
Same issue with creating a new account on first launch and trying to paste a generated password from a manger.

**Comment by swarna-sathya** [comment]
Same issue, both keyboard shortcut copy and "right-click-copy" do not work. It starts working on restarting the application. But after some time, it again stops working, not sure what triggers the issue.

**Comment by dkkeller** [comment]
Here's an AppleScript for the ones (like me) that don't want to type their mail and super long password
1) Replace `You@Mail.com` with your mail
2) Replace `YourSuperLongPassword` with your password
3) Run it (i.e. as an application with the Automator)
4) Wait for Franz to show up (navigate to the login screen) and click inside the user/mail text field (you may want to increase the `delay 8` if that takes too long)
5) Wait a few seconds 
6) Your login fields are filled and you're good to go
```
tell application "Franz"
	reopen
	activate
end tell
delay 8
tell application "System Events" to keystroke "You@Mail.com"
tell application "System Events" to keystroke tab
tell application "System Events" to keystroke "YourSuperLongPassword"
```

**Comment by rdewolff** [comment]
Same issue here. Anyone is investigating this issue?

**Comment by jancimajek** [comment]
Possible duplicate of #33

**Comment by rdewolff** [comment]
Yeah, not using Franz anymore due to that issue.

**Comment by artjazz111** [comment]
the same.
Do something. From 18th may 2018 you cant fix this issue. I'm premium user and i doint understand why you cant do this.

**Comment by bytasv** [comment]
@adlk are there any plans to address this issue? This is a really annoying one, something that forces to look for alternatives :(

**Comment by mitelg** [comment]
similar issue for me. Appeared in one of the latest 5.0.0-beta versions and is still not fixed in current stable 5.0.0
shortcuts like CTRL+C work for me, but right click -> copy doesn't

**Comment by marcolopes** [comment]
Same here... must use KEYBOARD shortcuts... those work!

**Comment by Razva** [comment]
Same here. Using the latest version, "mouse-copy" and "keyboard copy" don't work. For me this is a deal-breaker. :(

**Comment by pfunk1978** [comment]
Same on Ubuntu 18.04

**Comment by machinekoder** [comment]
Some here with Debian Stretch

**Comment by machinekoder** [comment]
Note: the drag and drop feature works from Franz to other apps, the copy-paste with keyboard from other apps to Franz. Still very annoying.
